lib/adal-angular.js

```diff
@@ -155,7 +155,9 @@ if (typeof module !== 'undefined' && module.exports) {
                                     $rootScope.$broadcast('adal:loginFailure', 'auto renew failure');
                                 } else {
                                     if (tokenOut) {
-                                        _oauthData.isAuthenticated = true;
+                                        $timeout(function () {
+                                            _oauthData.isAuthenticated = true;
```

Comment:
@nillis This isn't a scope change. In other places, we are making changes to rootscope, and hence timeout was needed.